suites/dream-challenge/synapaseLoginTest.js

```diff
@@ -0,0 +1,34 @@
+Feature('Synapse Login')
+
+const user = require('../../utils/user.js');
+const chai = require('chai');
+const {interactive, ifInteractive} = require('../../utils/interactive.js');
+const expect = chai.expect;
+const hostname = process.env.HOSTNAME;
+const profile = process.env.NAMESPACE;
+
+Scenario('Login with Google / Microsoft @manual', ifInteractive(
+    async(I) => {
+        const result = await interactive (`
+            1. Go to https://qa-brain.planx-pla.net
+            2. Check Dictionary button and check if graphic representation of data model is visible
+            3. Also check other button on the top menu bar
```

Comment:
should we be a bit more specific about this?

I assume `other button on the top menu bar` includes the nav bar buttons (dictionary, files, profile, etc...)?

if so, when tiered access is enabled, only the homepage and `/DD` link (dict viewer) are public, clicking on any other nav bar buttons should redirect user to login page